semantic_conventions/trace/messaging.yaml

```diff
@@ -180,3 +180,44 @@ groups:
         required:
           conditional: 'If missing, it is assumed to be false.'
         brief: 'A boolean that is true if the message is a tombstone.'
+  
+  - id: cloudevents
```

Comment:
The attributes from cloudevents are like `cloudevents.specversion`. To replicate this I had to add a new namespace `cloudevents`. I'm not sure what is the best approach. 

- Keep it as a new namespace to be the same as in the ones defined in cloudevents, 
- Add them under the `messaging` namespace. 

CloudEvents is a specification around events, so I'm not sure if makes sense to keep it under the namespace `messaging` on the same level as other messaging systems like Kafka.

Reply:
Let's go with the OTel conventions for OTel here. You can then discuss with CloudEvents maintainers if you should adapt the existing `keys.go` accordingly or introduce a separate one for OTel integrations.
Also make them aware of your semconv PR please.